src/components/Forms/CausesUpdateForm.vue

```diff
@@ -95,7 +93,7 @@
 					<strong>Your contribution will:</strong>
 				</div>
 				<div class="column">
-					<kv-select v-model="form.categoryId" class="group-dropdown">
+					<kv-select v-model="form.categoryId" class="group-dropdown" id="causes-categories-select">
```

Comment:
KvSelect requires id prop

Reply:
The goal with the required id is to help remind that all inputs need `<label>` elements. I think you could use the "Your contribution will", or make a new `<label class="tw-sr-only" for="causes-category-select">Some descriptive label of the select</label>` nearby